boards/z1/include/board.h

```diff
@@ -70,9 +70,9 @@ extern "C" {
  * @name    LED pin definitions and handlers
  * @{
  */
-#define LED0_PIN                    GPIO_PIN(4, 0)
-#define LED1_PIN                    GPIO_PIN(4, 1)
-#define LED2_PIN                    GPIO_PIN(4, 2)
+#define LED0_PIN                    GPIO_PIN(5, 4)
```

Comment:
according to [schematics](https://github.com/Zolertia/Resources/blob/master/Z1/Hardware/Revision%20C/Schematics/Zolertia%20Z1%20Schematics%20Revision%20C.pdf) LED1 is on P5.4, LED2 is on P5.6, and LED3 is on P5.5. 

Though we/RIOT could redefine the ordering, I still suggest to stick with vendor specs, hence IMHO it should be 
```
#define LED0_PIN                    GPIO_PIN(5, 4)
#define LED1_PIN                    GPIO_PIN(5, 6)
#define LED2_PIN                    GPIO_PIN(5, 5)
```

Reply:
Yes, you are correct. I did not really look at the schematics, but rather looked on the LED defines below (which are then also incorrect). Will fix.